lambdatimes/src/components/Content/Tabs.js

```diff
@@ -1,16 +1,36 @@
 import React from 'react';
 import Tab from './Tab';
+import PropTypes from 'prop-types';
+
+
 const Tabs = props => {
+
+
   return (
     <div className="tabs">
       <div className="topics">
         <span className="title">TRENDING TOPICS:</span>
         {/* map over the tabs provided on your props, create a new Tab component for each one.
             give the tab component a `selectTabHandler`, the `selectedTab`, and the `tab` itself as props*/}
+            {props.tabs.map(tab => {
+
+              return(
+              <Tab
+              key = {tab}
+              selectTabHandler = {props.selectTabHandler}
+              selectedTab = {props.selectedTab}
+              tab = {tab}
```

Comment:
Don't forget to tab these props 1 level deeper so it is easier to read and scroll through at a glance recognizing this is a component, these are its props tabbed over indicating nesting, and there is where it close at the same indent level as the opening of the nesting layer(in this case the `<Tab` and `/>`).